### PR TITLE
Tejas_work

### PR DESCRIPTION
**Pull request by Tejas-Rote**
Completed task Homepage - 06 
Implemented and also added Material UI library with custom theme.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,53 +1,64 @@
-import { CiWarning,CiMenuFries,CiUser,CiShop } from "react-icons/ci";
+import { CiWarning, CiMenuFries, CiUser, CiShop } from "react-icons/ci";
 import { HiOutlineShoppingBag } from "react-icons/hi";
+// import section6 from "./componenets/Homepage/Section6/section6";
+// import homePage from "./componenets/HomePage/homePage";
```
Comment (souravseal99) [inline src/App.js:4]: if these are redundant comments, if so please remove them.

File: src/App.js [hunk 2]
```diff
@@ -1,53 +1,64 @@
...
 import './App.css';
 
 function App() {
   return (
-   <div>
-      <nav class="main-nav">
+    <div>
+
+      <HomePage />
+      {/* <nav class="main-nav">
```
Comment (souravseal99) [inline src/App.js:14]: try to remove all the redundant comments here too... just keep the essential comments.

**Review by souravseal99 — CHANGES_REQUESTED** [review]

**Comment by souravseal99** [comment]
hey @Tejas-Rote, I have resolved the merge conflict and requested some changes.

File: src/App.js (hunk 1 above)
Comment (Tejas-Rote) [reply src/App.js:4]: ok

**Comment by Tejas-Rote** [comment]
please check .

**Comment by souravseal99** [comment]
any Updates @Tejas-Rote ?

**Comment by Tejas-Rote** [comment]
I merged branch into main also removed the comments

**Comment by souravseal99** [comment]
> I merged branch into main also removed the comments

great🥇

**Pull request merged**